setup-firewall/action.yaml

```diff
@@ -39,9 +39,13 @@ runs:
         # corresponding step part of the cleanup-cluster action.
         NETWORK="${{ inputs.cluster_name }}"
         NETWORK="${NETWORK//./-}"
+        # Name of the firewall rule, truncated at most at 61 according to gcloud sdk.
+        # We truncate the prefix since it is constant, trying to preserve as much as
+        # possible the provided input.
+        RULE_NAME=pod-cidrs-to-node-${NETWORK}
+        RULE_NAME=${RULE_NAME: (-61)}
 
-        gcloud compute firewall-rules create \
-          pod-cidrs-to-node-${NETWORK} \
+        gcloud compute firewall-rules create $RULE_NAME \
```

Comment:
These `firewall-rules` are later deleted by the `cleanup-cluster` action; mangling with the name here and not there would break that step: https://github.com/cilium/scale-tests-action/blob/pr%2Fgiorio94%2Fmisc/cleanup-cluster/action.yaml#L20-L25. 

While we could truncate the name there as well, I'd personally advocate for a simpler solution, just shortening the prefix enough so that it doesn't exceed the limits. Would `pods-to-node-` be enough? I assume so considering that `node-to-master-` is a few characters shorter.

Reply:
Thanks Marco, I didn't check the cleanup action :pray: 
Yes, shortening the name as you say would work, those `5` characters saved are just perfect.
Let me adjust commit.